assets/static/js/map/map.js

```diff
@@ -805,12 +805,12 @@
 
     function load_map() {
 
-        var southWest = L.latLng(11.57, 73.87),
-            northEast = L.latLng(18.45, 78.57),
+        var southWest = L.latLng(18.22, 81.56),
+            northEast = L.latLng(21.56, 86.46),
             bounds = L.latLngBounds(southWest, northEast);
```

Comment:
Figured out the `lat-long` for the southWest and northEast from http://www.mapsofindia.com/lat_long/orissa/.